lsr_role2collection.py

```diff
@@ -619,10 +619,10 @@ def process_readme(src_path, filename, rolename, original=None, issubrole=False)
 
 # Copy library, module_utils, plugins
 # Library and plugins are copied to dest_path/plugins
-# If plugin is in SUBDIR (currently, just module_utils),
-#   module_utils/*.py are to dest_path/plugins/module_utils/ROLE/*.py
-#   module_utils/subdir/*.py are to dest_path/plugins/module_utils/subdir/*.py
-SUBDIR = ("module_utils",)
+# If plugin is in SUBDIR (currently, modules and module_utils),
+#   {library,module_utils}/*.py are to dest_path/plugins/{modules,module_utils}/ROLE/*.py
+#   {library,module_utils}/subdir/*.py are to dest_path/plugins/{modules,module_utils}/subdir/*.py
+SUBDIR = ("library","module_utils",)
```

Comment:
as twoerner said, I think this should be done in the existing role, as a PR, and we shouldn't do this in the conversion

Reply:
Even creating a module_name subdir and moving a python module file into the subdir?
I thought renaming (renaming module_name to rolename_module_name) was not a good idea...
If that's the case, should we stop moving the module_utils?

Reply:
> Even creating a module_name subdir and moving a python module file into the subdir?

Yes.

> I thought renaming (renaming module_name to rolename_module_name) was not a good idea...

By renaming do you mean renaming in the script, or renaming in a PR?  The script should not rename, we should do the rename in a PR.

> If that's the case, should we stop moving the module_utils?

Yes.